core/src/main/java/com/netflix/iceberg/TableProperties.java

```diff
@@ -62,6 +62,9 @@
   public static final String SPLIT_LOOKBACK = "read.split.planning-lookback";
   public static final int SPLIT_LOOKBACK_DEFAULT = 10;
 
+  public static final String SPLIT_MIN_FILE_WEIGHT = "read.split.min-file-weight";
```

Comment:
Weight is called "size" in `read.split.target-size`, so I think we should make this similar. How about `read.split.min-size`?

Reply:
Wouldn't it imply that we cannot have a split smaller than `read.split.min-size`? It won't be true with the current logic as we use the new param only during bin packing.

I agree that we should change the name, especially because we are no longer limited to processing whole files. I am just thinking about the best option. We can also consider smth like `read.split.open-file-cost` and still keep the proposed logic, where split weight is `Math.min(length, openCost)`.

Reply:
You're right, that property is misleading. I like `read.split.file-open-cost`.